plugins/installer/override/override.php

```diff
@@ -337,11 +337,15 @@ private function saveOverrides($pks)
 		{
 			$insertQuery->clear('values');
 
-			$createdDate = $today->format('y-m-d h:i:s');
+			$tz = new \DateTimeZone(Factory::getApplication()->get('offset'));
+			$date = new Date('now');
+			$date->setTimezone($tz);
+			$createdDate = $date->toSql(true);
 
-			if (empty($pk->modifiedDate))
+			$pk->modifiedDate = '0000-00-00 00:00:00';
+			if (!empty($pk->modifiedDate))
```

Comment:
In this case, we never get   `$pk->modifiedDate` empty right? So, modifiedDate is always `$createdDate`?
In without this PR or previous. When the core file is removed not updated. We did not get modified date because core file is removed right ?
Then, we show the in override list that `core file removed`.
So, i think just add `$pk->modifiedDate = '0000-00-00 00:00:00';` after if statment . Then, its fine ?